### DISCUSSION TITLE
Quickly numbers numbering branches and leaves for task implementation order.

### DISCUSSION THREAD

**Discussion by automagicmike** [freeplane/freeplane#1844] [Help] [answered]
Hi
Is there any way to quickly give branches and leaves numbers?
e.g. in Number mode clicking on a branch gives decorates a branch/leaf with the next sequential number.

This is because once I've created a mm to organise my thoughts the actually implementation order might look different.
Can anyone suggest approaches they have used

Thanks

**Comment by macmarrum** [comment]
Hi @automagicmike 

Could you please illustrate what you need, preferably using an example mind map (you can ZIP it and attach it to a post).

**Reply by automagicmike** [reply]
Hi
An example.
Style isn't really important.
It would be more about numbering quickly and amending that order if required.

![mind-map-numbering](https://github.com/freeplane/freeplane/assets/442584/8408246e-2ed5-45c9-a732-2c51babf8cb0)

**Comment by dpolivaev** [comment, marked as answer]
<img width="780" alt="nodeNumbering" src="https://github.com/freeplane/freeplane/assets/1558052/ecabb510-2a84-40c7-bb9e-fa10797fa9a4">

**Reply by dpolivaev** [reply]
You can activate it for any style

**Comment by automagicmike** [comment]
Dimitry

Thanks for your help.
In the process of looking for that toolbar in your answer
(Right Mouse Menu -> Tool Panel) 
I discovered icons !

Ctrl F2 and a quick selection - this works really well for me as
it  is quick and can number nodes non sequentially and
even add extra icons - fantastic.

Thanks once again
![mm2a](https://github.com/freeplane/freeplane/assets/442584/b3320a8d-c8bf-4dc0-bef4-fcf4a477c2e5)

Mike
